sdk/eventhub/event-hubs/test/snippets.spec.ts

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  EventHubProducerClient,
+  EventHubConsumerClient,
+  earliestEventPosition,
+} from "../src/index.js";
+import { DefaultAzureCredential } from "@azure/identity";
+// @ts-ignore
+import { ContainerClient } from "@azure/storage-blob";
```

Comment:
I am not expecting these ts-ignores, why are the needed?

Reply:
Because there is no direct reference to the @azure/storage-blob, so in this case we're assuming it is there just for documentation purposes.